common/resource/resource.go

```diff
@@ -85,6 +86,7 @@ type (
 		GetMatchingClient() matching.Client
 		GetHistoryRawClient() history.Client
 		GetHistoryClient() history.Client
+		GetRatelimiterAggregatorsClient() rpc.Client
```

Comment:
this is probably the main "maybe I should rename that package" thing to me :|
any naming thoughts?

Reply:
that seems clear to me as is imho